Library/Formula/i2p.rb

```diff
@@ -0,0 +1,25 @@
+class I2p < Formula
+  desc "I2P is an anonymous overlay network - a network within a network"
+  homepage "https://geti2p.net"
+  url "https://download.i2p2.de/releases/0.9.21/i2pinstall_0.9.21.jar"
+  sha256 "0238ffc6ea44099ef4fe6c20913cb4eec675c2760aea07dfe7d499addcc89cf2"
+
+  depends_on :java => "1.6+"
+
+  def install
+    open("i2p_template.conf", "w") do |f|
+      f.write("INSTALL_PATH=#{prefix}")
+    end
+
+    system "java -jar i2pinstall_0.9.21.jar -options i2p_template.conf"
+    system "ln -s #{prefix}/i2psvc-macosx-universal-64 #{prefix}/i2psvc"
```

Comment:
You don't need to call `system` on any of these. Ruby's FileUtils can handle `ln`, `mv`, `mkdir` and so on. You can actually avoid them all together by doing something like:

```ruby
libexec.install_symlink libexec/"i2psvc-macosx-universal-64" => "i2psvc"
```